Build scripts for better-ajv-errors

Mahima-Gaikwad opened the pull request:
## Checklist
<!--- Goto Preview tab for better readability -->
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

seth-priya commented on b/better-ajv-errors/LICENSE:
```diff
@@ -0,0 +1,13 @@
+Copyright 2018 Atlassian Pty Ltd
```
this is the script license and not the package license, can just be copied over from the base folder of this repo

Mahima-Gaikwad replied:
Updated license file

seth-priya commented on b/better-ajv-errors/better-ajv-errors_ubi8.4.sh:
```diff
@@ -0,0 +1,49 @@
+# Tested on     : UBI 8.4
+# Script License: Apache 2.0 License
+# Maintainer    : Mahima Gaikwad <mahima.gaikwad@ibm.com>
+#
```
Please add the additional fields that have been introduced recently - Language, Travis-Check etc

Mahima-Gaikwad replied:
Added Language, Travis-Check fields

seth-priya commented on b/better-ajv-errors/better-ajv-errors_ubi8.4.sh:
```diff
@@ -0,0 +1,49 @@
+#Variables
+PACKAGE_URL=https://github.com/atlassian/better-ajv-errors.git
+PACKAGE_VERSION=v0.5.7
+PACKAGE=better-ajv-errors
```
use PACKAGE_NAME for better clarity

Mahima-Gaikwad replied:
used PACKAGE_NAME

Mahima-Gaikwad replied:
used PACKAGE_NAME

seth-priya commented on b/better-ajv-errors/better-ajv-errors_ubi8.4.sh:
```diff
@@ -0,0 +1,49 @@
+yum update -y 
+
+#Install dev dependencies 
+yum install -y git wget gcc gcc-c++ python2 make
```
are all of these dependencies needed?

Mahima-Gaikwad replied:
Dependencies not needed, removed

seth-priya commented on b/better-ajv-errors/better-ajv-errors_ubi8.4.sh:
```diff
@@ -0,0 +1,49 @@
+NODE_VERSION=v12.22.8
+DISTRO=linux-ppc64le
+
+yum update -y 
```
do we need a yum update?

Mahima-Gaikwad replied:
Removed yum update

seth-priya commented on c/call-bind/LICENSE:
```diff
@@ -0,0 +1,21 @@
+MIT License
```
same comment as above - the LICENSE can just be copied over from the base folder

seth-priya commented on c/call-bind/call-bind_ubi8.4.sh:
```diff
@@ -0,0 +1,51 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : call-bind
+# Version       : master(a994df69f401f4bf735a4ccd77029b85d1549453)
```
let's say Commit # : <commit id>

Mahima-Gaikwad replied:
Updated commit id

seth-priya commented on c/call-bind/call-bind_ubi8.4.sh:
```diff
@@ -0,0 +1,51 @@
+# Version       : master(a994df69f401f4bf735a4ccd77029b85d1549453)
+# Source repo   : https://github.com/ljharb/call-bind.git
+# Tested on     : UBI 8.4
+# Script License: MIT License
```
this is script license, not the package license, should be Apache 2.0

Mahima-Gaikwad replied:
Updated License

seth-priya commented on c/call-bind/call-bind_ubi8.4.sh:
```diff
@@ -0,0 +1,51 @@
+
+#Variables
+PACKAGE_URL=https://github.com/ljharb/call-bind.git
+PACKAGE=call-bind
```
use "PACKAGE_NAME" for clarity

seth-priya commented on c/call-bind/call-bind_ubi8.4.sh:
```diff
@@ -0,0 +1,51 @@
+yum update -y 
+
+#Install dev dependencies 
+yum install -y git wget gcc gcc-c++ python2 make
```
are all of these needed?

seth-priya commented on c/call-bind/call-bind_ubi8.4.sh:
```diff
@@ -0,0 +1,51 @@
+NODE_VERSION=v12.22.8
+DISTRO=linux-ppc64le
+
+yum update -y 
```
is this needed?

seth-priya commented:
Generic question - any specific reason for not checking " Have you checked and followed all the points mention in the CONTRIBUTING.MD" above?

Mahima-Gaikwad commented:
Updated builds scripts as per above comments.
Updated License file, replaced package License with script License.
Checked "Have you checked and followed all the points mention in the CONTRIBUTING.MD"
